lib/que/version.rb

```diff
@@ -4,6 +4,6 @@ module Que
   VERSION = '1.3.1'
 
   def self.job_schema_version
-    1
+    2
```

Comment:
Can line 4 above this be changed to `2.0.0.beta1` or similar? I am writing a change for que-scheduler to work with these code changes, and it will be nice to target this version by number.

Reply:
Needed for this corresponding change: https://github.com/hlascelles/que-scheduler/pull/328

Reply:
Done @hlascelles!